tests/datasets/test_datasets.py

```diff
@@ -109,6 +110,30 @@ def test_blogcatalog3_deprecated_load() -> None:
         load_dataset_BlogCatalog3(dataset.data_directory)
 
 
+def test_mutag_load() -> None:
+    graphs, labels = MUTAG().load()
+
+    n_graphs = 188
+
+    assert len(graphs) == n_graphs
+    assert len(labels) == n_graphs  # one label per graph
+
+    # get a list with the number of nodes in each graph
+    n_nodes = [g.number_of_nodes() for g in graphs]
+
+    # calculate average and max number of nodes across all graphs
+    n_avg_nodes = np.mean(n_nodes)
+    max_nodes = np.max(n_nodes)
+
+    # average number of nodes should be 17.93085... or approximately 18.
+    assert round(n_avg_nodes) == 18
```

Comment:
Maybe instead of looking at the the averages, we could look at something exact:

```suggestion
    assert sum(n_nodes) == 3371
```

And similarly, verify that the edge selection is working appropriately, because it's slightly subtle:


```python
n_edges = [g.number_of_edges() for g in graph]
assert sum(n_edges) == 7442
````